service/profiles/spp/spp_service.c

```diff
@@ -652,16 +678,14 @@ static void spp_proxy_connection_callback(euv_pipe_t* handle, int status, void*
         return;
     }
 
+    spp_rx_buffer_send(device); /* send cached received data */
```

Comment:
it mayebe in SPP_PROXY_STATE_CLOSING state, we do not need to do spp_rx_buffer_send